admin-dev/themes/new-theme/js/bundles/csaguzzle

```diff
@@ -0,0 +1 @@
+/var/www/html/prestashop/vendor/csa/guzzle-bundle/src/Resources/public
```

Comment:
What are these files ? 🤔

Reply:
Im not sure, it was generated with `fosjsrouting` after command `bin/console assets:install`

Reply:
Must be removed because depending on your environment.